x-pack/plugin/core/template-resources/src/main/resources/ml/config_index_mappings.json

```diff
@@ -443,6 +443,9 @@
       "model_snapshot_retention_days" : {
         "type" : "long"
       },
+      "platform_architecture" : {
+        "type" : "keyword"
+      },
```

Comment:
D'oh, I told you the wrong file to put this in. It needs to go in `inference_index_mappings.json`.

Reply:
How do we know where in that file it needs to go?

Reply:
That file isn't in alphabetical order, so just put it quite near the top similar to what we did in `toXContent`, after `model_id`.